core/templates/dev/head/pages/exploration_editor/translation_tab/translator_overview_directive.html

```diff
@@ -14,7 +14,7 @@
     <div class="progress oppia-translation-progess-bar">
       <div class="progress-bar progress-bar-success" role="progressbar" ng-style="getTranslationProgressStyle()"></div>
     </div>
-    <span class="oppia-numerical-progress-status protractor-test-translation-numerical-status">(<[numberOfRequiredAudio - numberOfAudioNotAvailable]>/<[numberOfRequiredAudio]>)</span>
+    <span aria-label="<[numberOfRequiredAudio - numberOfAudioNotAvailable]> items translated out of <[numberOfRequiredAudio]> items" role="status" class="oppia-numerical-progress-status protractor-test-translation-numerical-status">(<[numberOfRequiredAudio - numberOfAudioNotAvailable]>/<[numberOfRequiredAudio]>)</span>
```

Comment:
That's not exactly what I meant. The text should read "1 item out of 8 items" if numberOfRequiredAudio - numberOfAudioNotAvailable == 1 else it should read "x items out of 8 items".

Reply:
Done